akka-persistence/src/test/scala/akka/persistence/PersistentActorRecoveryTimeoutSpec.scala

```diff
@@ -130,18 +128,24 @@ class PersistentActorRecoveryTimeoutSpec
       system.stop(persisting)
       expectTerminated(persisting)
 
-      // now replay, but don't give the journal any tokens to replay events
```

Comment:
at least this comment didn't match with the implementation, and I didn't understand what we tried to test with a long receive timeout and not triggering the recovery timeout